OpenConsole.sln

```diff
@@ -1192,6 +1180,7 @@ Global
 		{376FE273-6B84-4EB5-8B30-8DE9D21B022C}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{376FE273-6B84-4EB5-8B30-8DE9D21B022C}.Debug|ARM64.ActiveCfg = Debug|Any CPU
 		{376FE273-6B84-4EB5-8B30-8DE9D21B022C}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{376FE273-6B84-4EB5-8B30-8DE9D21B022C}.Debug|x64.Build.0 = Debug|Any CPU

```

Comment:
Why are there changes to this file at all?

Reply:
I have no idea -- visual poodio just added them for some reason best known to itself. I only noticed this now. I will filter out the changes.